### PR TITLE
Added bottom to up button in the index file.

### PR DESCRIPTION
**Pull request by aashimgarg**
Fixed issue : #76 
The bottom to up button is added with easy css,js,html. This is very helpful as it helps to go to top of the page with just clicking a button.

### CODE REVIEW

**Comment by aashimgarg** [comment]
@jobara ?

**Comment by jobara** [comment]
@aashimgarg I mentioned on the Issue that we need to think about the design more. First to determine if it is something that is needed for this site, and, if so, what the intended design for it is.

File: css/button.css [hunk 1]
```diff
@@ -0,0 +1,27 @@
+
```
Comment (Chaitanyassr) [inline css/button.css]:
```suggestion
changes should be minimum.
```

**Comment by sachin10101998** [comment]
Stale and inactive PR. It requires more discussion. May or may not be closed.

**Comment by greatislander** [comment]
This PR won't be merged as we are currently beginning a redesign process. Thanks for your contribution!

**Pull request closed without merging**